packages/web/app/components/Surveys/SurveyScreenPaginator.jsx

```diff
@@ -101,6 +103,7 @@ export const SurveyScreenPaginator = ({
         status={status}
         setStatus={setStatus}
         showCancelButton={showCancelButton}
+        encounterType={encounter?.encounterType}
```

Comment:
@dannash100 this is a refactor to pass in encounter type so I didn't have to move useEncounter to ui-compoennts